src/store/keyboard/context.js

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext } from 'react';
+
+const keyboardContext = createContext();
```

Comment:
Can you please capitalize the initial letter of `keyboardContext` here ?

Reply:
will do that